pkg/operator/ceph/version/version.go

```diff
@@ -106,7 +112,17 @@ func ExtractCephVersion(src string) (*CephVersion, error) {
 		return nil, fmt.Errorf("failed to parse version extra part: %s", m[2])
 	}
 
-	return &CephVersion{major, minor, extra}, nil
+	mm := buildVersionPattern.FindStringSubmatch(src)
+	if mm == nil {
+		return nil, fmt.Errorf("failed to parse version from: %q", src)
```

Comment:
wait, upstream images won't have this build version set so we need to ignore this error, right?

Reply:
fixed!